portfolio/src/main/webapp/index.html

```diff
@@ -8,21 +8,25 @@
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
+      <h1>CJ's Portfolio</h1>
       <a href="https://github.com/cjhbh3" target="_blank">Github</a>
+      <h2>About Me</h2>
       <p>
-          Hi, my name is CJ. I just finished my freshmen year at Missour S&T.
-          I was born in Indiana and now live in Missouri. 
-          I have worked in many IT jobs.
-          My love for computer programming begain freshmen year of high school
-          when I became the lead programmer for my high school's FTC Robotics team.
+        Hi, my name is CJ.</br>
+        I just finished my freshmen year at Missour S&T.</br>
+        I was born in Indiana and now live in Missouri.</br>
+        I have worked in many IT jobs.</br>
+        Why did I start CS? My experience as a lead programmer in FTC Robotics during High School.</br>
       </p>
       <p>This is my dog, Juliette.</p>
       <img src="/images/Juliette.jpg" width="500" height="500">
       <p>Here is what I look like!</p>
       <img src="/images/SelfPic.JPG" with="500" height = "500">
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
+      <p>Fetching Text</p>
+      <button onclick="getRandomQuoteUsingArrowFunctions()">Fetch Me that Text</button>
```

Comment:
Is the goal of this "Fetch Me that Text" button to print "Hello CJ!" or add a new quote to the container of quotes that would be shown on the page when the "Give me a Quote" button is clicked?

The website currently prints "Hello CJ!" when the "Fetch me that Text" button is clicked due to the doGet from DataServlet.java.

Reply:
As far as I understood from the tutorial, the goal of the fetch() function was to grab the text from the /data page and print it. In my website, it should print "Hello CJ!".